plugin/trino-hive/src/main/java/io/trino/plugin/hive/fs/CachingDirectoryLister.java

```diff
@@ -244,9 +252,10 @@ public long getRequestCount()
     }
 
     @Override
-    public boolean isCached(Location location)
+    public boolean isCached(Location location, SchemaTableName schemaTableName)
     {
-        ValueHolder cached = cache.getIfPresent(location);
+        CacheKey cacheKey = new CacheKey(location, schemaTableName);
```

Comment:
Nit: no need to store a local variable here since the result is only used to pass into the `cache.getIfPresent` call

Reply:
done